packages/smithy-client/src/date-utils.ts

```diff
@@ -33,3 +36,232 @@ export function dateToUtcString(date: Date): string {
 
   return `${days[dayOfWeek]}, ${dayOfMonthString} ${months[month]} ${year} ${hoursString}:${minutesString}:${secondsString} GMT`;
 }
+
+const daysInMonth = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
```

Comment:
I was matching the style of the file previously (like days and months, above). I'll go ahead and change all constants to be consistent.

Reply:
nit: might move line just below `const months` since the order is related. Or just before `const validateDayOfMonth` if you want to keep it closer to where it is used.